www/install/php/Update-22.04.2.php

```diff
@@ -1,20 +0,0 @@
-<?php
```

Comment:
this file must not change, since it's already in stable version : https://github.com/centreon/centreon/blob/22.04.x/www/install/php/Update-22.04.2.php

Reply:
@kduret done :)